lib/menu.rb

```diff
@@ -0,0 +1,19 @@
+require_relative 'dish'
+
+class Menu
+
+  attr_reader :dishes
+
+  def initialize
+    @dishes = []
+  end
+
+  def add_dish(dish)
```

Comment:
Metrics/AbcSize: Assignment Branch Condition size for add_dish is too high. [17.09/15]